index.html

```diff
@@ -3405,7 +3405,10 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
     relate things of different types, where the vocabularies in use within
     different entities calls for different context scoping. For example,
     <code>hasPart</code>/<code>partOf</code> may be common terms used in a document, but mean
-    different things depending on the context.</p>
+    different things depending on the context.
+    A <a>context</a> scoped on <code>@type</code> is only in effect for the <a>node object</a> on which
+    the type is used; the previous in-scope <a>contexts</a> are placed back into
+    effect when traversing into another <a>node object</a>.</p>
```

Comment:
To be precise, we should add something like "amended by the appropriate property-scoped context and/or local context, if any." But I'm affraid that might confuse the reader more than anything else...

Reply:
Perhaps: `Note that any property-scoped or local contexts that were introduced in the node object would still be in effect when traversing into another node object.`

Reply:
Applied @dlongley's suggestion.